Parsers/ASimUserManagement/ASimUserManagementLinuxAuthpriv.yaml

```diff
@@ -0,0 +1,342 @@
+      | extend 
+          split(Temp_GroupName, "/")
+      | extend 
+          GroupName = tostring(Temp_GroupName[0]),
+          GroupId   = tostring(Temp_GroupName[1])
+      | project-away Temp_GroupName
+      | extend 
+          EventType   = "GroupModified",
+          EventResult = "Success"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupmod"
+          and SyslogMessage startswith "failed to change "
+      | parse SyslogMessage with "failed to change " * " (group " Temp_GroupName ", new name: " *
+      | extend split(Temp_GroupName, "/")
+      | extend 
+          GroupName = tostring(Temp_GroupName[0]),
+          GroupId   = tostring(Temp_GroupName[1])
+      | project-away Temp_GroupName
+      | extend 
+          EventType   = "GroupModified",
+          EventResult = "Failure"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "groupdel"
+      | parse SyslogMessage with "group '" GroupName "' removed" *
+      | extend 
+          EventType   = "GroupDeleted",
+          EventResult = "Success"
+      | invoke ItemParser()
+  ),(
+      Syslog
+      | where not(disabled)
+      | where Facility == "authpriv"
+      | where ProcessName == "gpasswd"
+      | parse SyslogMessage with "user " TargetUsername " " Action " by " ActorUsername " " * " group " GroupName
+      | lookup ActionLookup on Action
+      | project-away Action
+      | extend 
+          EventResult = "Success"
+      | invoke ItemParser()
+  )
+  | where not(disabled)
+  | invoke _ASIM_ResolveDvcFQDN ("Computer")
+  | lookup SeverityLookup on SeverityLevel
+  | project-rename 
+      ActiveAppName = ProcessName,
+      DvcId         = _ResourceId
+  | extend
+      EventCount          = int(1),
+      EventEndTime        = TimeGenerated,
+      EventProduct        = 'authpriv',
+      EventSchema         = "UserManagement",
+      EventSchemaVersion  = "0.1.1",
+      EventSeverity       = "Informational",
+      EventStartTime      = TimeGenerated,
+      EventVendor         = "Linux",
+      ActingAppType       = "Process",
+      Hostname            = DvcHostname,
+      DvcIdType           = iff (DvcId == "", "", "AzureResourceID"),
+      DvcIpAddr           = iif(HostIP == "Unknown IP","",HostIP),
+      DvcOs               = "Linux",
+      UpdatedPropertyName = EventSubType,
+      ActingAppId         = tostring(ProcessID),
+      TargetUserIdType    = iif(isnotempty(TargetUserId), "UID", ""),
+      TargetUsernameType  = iif(isnotempty(TargetUsername), "Simple", ""),
+      ActorUsernameType   = iif(isnotempty(ActorUsername), "Simple", ""),
+      GroupIdType         = iif(isnotempty(GroupId), "UID", ""),
+      GroupNameType       = iif(isnotempty(GroupName), "Simple", ""),
+      User                = ActorUsername
```

Comment:
following are recommended fields, can we map them?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/64053d54-15f3-40d6-800d-ceaacc1e6a4b)

Reply:
I do not have any in my dataset, will validate soon.

Reply:
Verified but no found no source data